qa/suites/rgw/lua/supported-all-distro$/ubuntu_latest.yaml

```diff
@@ -0,0 +1 @@
+../all/ubuntu_latest.yaml
```

Comment:
this looks like a broken symlink. we shouldn't create a `supported-all-distro$` subdirectory and link things into it. just replace that subdirectory with a symlink to `.qa/distros/supported-random-distro$` like our other subsuites do